app/src/androidTest/java/net/skyscanner/backpack/BpkTestVariant.kt

```diff
@@ -76,7 +65,10 @@ sealed class BpkTestVariant(val id: String) {
       if (Looper.myLooper() == null) {
         Looper.prepare()
       }
-      val dialog = AppCompatDialog(super.newContext(context))
+      val config = Configuration(context.resources.configuration).apply {
```

Comment:
Why we did that?

Reply:
Without this the context didn't have the right configuration and wasn't using night mode correctly - I'm not sure how it was working before tbh - for the rtl config the `newContext` method was also setting the properties